frame/collective/src/benchmarking.rs

```diff
@@ -529,7 +527,7 @@ benchmarks_instance_pallet! {
 	}
 
 	close_approved {
-		let b in 1 .. MAX_BYTES;
+		let b in 2 .. MAX_BYTES;
```

Comment:
Why these changes?

Reply:
The goal of this PR was to be able to set the `MaxProposal` to a value higher than `255`. In the benchmarks, we use the `remark` function to represent a proposal. Because we cannot have two identical proposals what we did until now is make the remark data be a variable number repeated a number of times(`b` times). This comes with a limit since the remark data is a `Vec<u8>` so it cannot accept a number greater than `255`. To fix this we use little-endian to encode these greater numbers. So to represent a number greater than `255` we need a vector that is longer than 1. That is why we updated the range of  `b` so that it starts from 2.

Reply:
Then it should be changed to `4`, because `MaxMembers` is an `u32` which would lead to duplicate proposals when using more than 65k max members. You should also leave some comment on why we start at 4.

Reply:
https://github.com/paritytech/substrate/pull/12454#discussion_r994730425

Reply:
Hmm okay :see_no_evil:

Reply:
I'm not happy with this, but okay :P